### DISCUSSION TITLE
南北极空洞问题

### DISCUSSION THREAD

**Discussion by zhr5422** [zouyaoji/vue-cesium#543] [Q&A]
从原来的老项目迁移过来的，用的自己的  imageryProvider 和terrainProvider，目前就定位到，只要注释掉imageryProvider 南北极就没有空洞了

**Comment by zouyaoji** [comment]
我知道的是3857坐标系南北极就有空洞。确认imageryProvider 是否加载了3857坐标系的数据。

如果确认是4326，vue-cesium加出来有空洞，而原生没空洞，麻烦提供vue-cesium和原生加载的代码，以助于我分析解决问题。

**Reply by zhr5422** [reply]
imageryProvider: new Cesium.UrlTemplateImageryProvider(
{ url:xxx.jpg,
  tilingScheme: new Cesium.GeographicTilingScheme({ellipsoid: Cesium.Ellipsoid.WGS84})


这段代码是从老项目复制过来的

**Reply by zhr5422** [reply]
url： map/gmap/z/x/y.jpg

**Reply by zhr5422** [reply]
![image](https://github.com/zouyaoji/vue-cesium/assets/22064266/634b8aa8-5743-4b74-92ef-c501d741db3e)

**Comment by zhoubonet** [comment]
这个问题是否已经解决

**Reply by zhoubonet** [reply]
4490的坐标系也有这个问题，怎么解决啊

**Reply by zhr5422** [reply]
> 这个问题是否已经解决

问题出在 imageryLayer以及Cesium对象的使用上。应该在viewer ready 事件里手动添加layer，使用 ready 事件里给的Cesium 进行初始化ImageryLayer实例
